src/cli/cli.ts

```diff
@@ -981,7 +981,7 @@ async function handleMultipleResultsFound<T>(message: string, values: { [key: st
     throw new Error(`${message} Found: ${Object.keys(values).join(', ')}.`);
   }
 
-  cli.error(`🌶️  ${message} `);
+  cli.error(`🌶️ ${message} `);
```

Comment:
Hi @milanholemans, this is not part of the functionality addressed in the PR. I'm making a point of this because in my powershell terminal, removing the extra space causes no space to be rendered at all:

![alt text](image.png)

I agree that we should do something about this, even it it's minor. But let's not do it in this PR and see if we can find a way to fix this right for all shells.